com.unity.multiplayer.mlapi/THIRD_PARTY_NOTICES.md

```diff
@@ -1,6 +1,17 @@
-MIT License
+This package contains third-party software components governed by the license(s) indicated below:
```

Comment:
For some reason Git thinks this was a file move not a new third party file, probably because I deleted the LICENSE.md file from the root dir per @wackoisgod's guidance